redisgears_core/src/noop/mod.rs

```diff
@@ -0,0 +1,350 @@
+                encver,
+                REDIS_GEARS_VERSION
+            );
+            return raw::REDISMODULE_ERR as i32;
+        }
+
+        let inner_ctx = unsafe { raw::RedisModule_GetContextFromIO.unwrap()(rdb) };
+        let ctx = Context::new(inner_ctx);
+
+        match aux_load_internals(&ctx, rdb) {
+            Ok(_) => raw::REDISMODULE_OK as i32,
+            Err(e) => {
+                log::warn!("Failed loading functions from rdb, {}.", e);
+                raw::REDISMODULE_ERR as i32
+            }
+        }
+    }
+}
+
+mod commands {
+    use redis_module::{RedisResult, RedisString, RedisValue};
+    use redis_module_macros::command;
+
+    use super::*;
+
+    #[command(
+    {
+        name: "tfcall",
+        flags: [MayReplicate, DenyScript, NoMandatoryKeys],
+        arity: -3,
+        key_spec: [
+            {
+                flags: [ReadWrite, Access, Update],
+                begin_search: Index({ index : 2}),
+                find_keys: Keynum({ key_num_idx : 0, first_key : 1, key_step : 1 }),
+            }
+        ],
+    }
+)]
+    fn function_call(_ctx: &Context, _args: Vec<RedisString>) -> RedisResult {
+        RedisResult::Ok(RedisValue::Null)
+    }
+
+    #[command(
+    {
+        name: "tfcallasync",
+        flags: [MayReplicate, DenyScript, NoMandatoryKeys],
+        arity: -3,
+        key_spec: [
+            {
+                flags: [ReadWrite, Access, Update],
+                begin_search: Index({ index : 2}),
+                find_keys: Keynum({ key_num_idx : 0, first_key : 1, key_step : 1 }),
+            }
+        ],
+    }
+)]
+    fn function_call_async(_ctx: &Context, _args: Vec<RedisString>) -> RedisResult {
+        RedisResult::Ok(RedisValue::Null)
+    }
+
+    #[command(
+    {
+        name: "_rg_internals.function",
+        flags: [MayReplicate, DenyScript, NoMandatoryKeys],
+        enterprise_flags: [ProxyFiltered],
+        arity: -3,
+        key_spec: [],
+    }
+)]
+    fn function_command_on_replica(_ctx: &Context, _args: Vec<RedisString>) -> RedisResult {
+        RedisResult::Ok(RedisValue::Null)
+    }
+
+    #[command(
+    {
+        name: "tfunction",
+        flags: [MayReplicate, DenyScript, NoMandatoryKeys],
+        arity: -2,
+        key_spec: [],
+    }
+)]
+    fn function_command(_ctx: &Context, _args: Vec<RedisString>) -> RedisResult {
+        RedisResult::Ok(RedisValue::Null)
+    }
+
+    #[command(
+    {
+        name: "_rg_internals.update_stream_last_read_id",
+        flags: [MayReplicate, DenyScript],
+        enterprise_flags: [ProxyFiltered],
+        arity: 6,
+        key_spec: [
+            {
+                flags: [ReadWrite, Access, Update],
+                begin_search: Index({ index : 3}),
+                find_keys: Range({ last_key: 0, steps: 1, limit: 0 }),
+            }
+        ],
+    }
+)]
+    fn update_stream_last_read_id(_ctx: &Context, _args: Vec<RedisString>) -> RedisResult {
+        RedisResult::Ok(RedisValue::Null)
```

Comment:
This is an internal command between to replica and AOF, it should return a simple OK reply